### PR TITLE
Update notes for getting started for internal folks

### PR DESCRIPTION
**Pull request by BryanFauble**
**Problem:**
In the contributing.md guide we were blanket mentioning to fork the repo and work from there.

**Solution:**
I updated the docs to mention the flow for internal/external folks contributing to the repo

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ By contributing, you are agreeing that we may redistribute your work under this
 - [How to contribute](#how-to-contribute)
    * [Reporting bugs or feature requests](#reporting-bugs-or-feature-requests)
 - [Getting started](#getting-started)
-    + [Fork and clone this repository](#fork-and-clone-this-repository)
+    + [Internal sage collaborators: Clone this repository](#internal-sage-collaborators-clone-this-repository)
```
Comment (jaymedina) [inline CONTRIBUTING.md:11]:
```suggestion
    + [Internal Sage collaborators: Clone this repository](#internal-sage-collaborators-clone-this-repository)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -48,7 +49,22 @@ Bug reports and feature requests can be made in two ways. The first (preferred)
 After a bug report is received, expect a Sage Bionetworks staff member to contact you through the submission method you chose ([Synapse Help Forum](https://www.synapse.org/#!SynapseForum:default)or [Github issue](https://github.com/Sage-Bionetworks/synapsePythonClient/issues)). After ascertaining there is enough detail for the bug report or feature request, a JIRA issue will be opened. If you want to work on fixing the issue or feature yourself, follow the next sections!
 
 ## Getting started
-#### Fork and clone this repository
+#### Internal sage collaborators: Clone this repository
```
Comment (jaymedina) [inline CONTRIBUTING.md:52]:
```suggestion
#### Internal Sage collaborators: Clone this repository
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -48,7 +49,22 @@ Bug reports and feature requests can be made in two ways. The first (preferred)
...
+For internal sage collaborators you will have access and permissions to create branches
```
Comment (jaymedina) [inline CONTRIBUTING.md:53]:
```suggestion
For internal Sage collaborators, you will have access and permissions to create branches
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -48,7 +49,22 @@ Bug reports and feature requests can be made in two ways. The first (preferred)
...
+within the central repoistory for this project. As a result instead of creating a fork
```
Comment (jaymedina) [inline CONTRIBUTING.md:54]:
```suggestion
within the central repository for this project. As a result, instead of creating a fork
```

**Review by jaymedina — APPROVED** [review]
Hey @BryanFauble , thanks for the quick work on this and making things more explicit. I added some small suggestions. Very minor, so feel free to ignore most (except for that one typo fix). Aside from that, this is already in good shape, so I'll approve.

Thanks!

**Comment by BryanFauble** [comment]
Excellent! Thanks much @jaymedina

**Review by danlu1 — APPROVED** [review]
LGTM

**Pull request merged**